Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -1,4 +1,4 @@
-// Generated by Apple Swift version 5.6.1 (swiftlang-5.6.0.323.66 clang-1316.0.20.12)
+// Generated by Apple Swift version 5.7 (swiftlang-5.7.0.123.7 clang-1400.0.29.50)
```

Comment:
Can somebody not running macOS Ventura re-compile this with Xcode 13? 🙏🏻

Reply:
On it

Reply:
No rush though, we should probably wait for Monday?

Reply:
Yeah, I think we should wait to release on Monday, I'm on call so I'm here for this type of thing 😄